### PR TITLE
fix(alerts and ai): updated baselines and anomalies

### PR DESCRIPTION
**Pull request by homelessbirds**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ redirects:
   - /docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-new-relic-ai
 ---
 
-With applied intelligence's proactive detection, anomalies from your APM-monitored applications are automatically surfaced in our activity streams and anomalies feed. You can click each anomaly to bring up an automatic analysis.
+With applied intelligence's anomlay detection, anomalies from your APM-monitored applications are automatically surfaced in our activity streams and anomalies feed. You can click each anomaly to bring up an automatic analysis.
```
Comment (austin-schaefer) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx:14]: Looks like there's a typo on this one.

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/set-thresholds-alert-condition.mdx [hunk 2]
```diff
@@ -77,7 +77,7 @@ Thresholds are set during the process of [creating a condition](/docs/alerts/new
   </tbody>
 </table>
 
-To learn more about specific alert condition types (like baseline and NRQL), see [Types of conditions](/docs/alerts/new-relic-alerts/defining-conditions/define-alert-conditions#condition-types).
+To learn more about specific alert condition types (like anomaly and NRQL), see [Types of conditions](/docs/alerts/new-relic-alerts/defining-conditions/define-alert-conditions#condition-types)..
```
Comment (austin-schaefer) [inline src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/set-thresholds-alert-condition.mdx:80]: Looks like there's a double period at the end of this sentence

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/other-condition-types/create-baseline-alert-conditions.mdx [hunk 3]
```diff
@@ -26,92 +26,92 @@ import baselineOneFacet from 'images/baseline-one-facet.png'
...
 * Dynamically adjust to changing data and trends, including daily or weekly trends.
 
-In addition, baseline alerting works well with new applications when you do not yet have known behaviors.
+In addition, anomaly alerting works well with new applications when you do not yet have known behaviors.
 
-## How it works [#baseline-overview]
+## How it works [#anomaly-overview]
 
-When you choose a data source (for example, an APM metric) for a baseline condition, we'll use the past values of that data to dynamically predict the data's near-future behavior.
+When you choose a data source (for example, an APM metric) for an anomaly  condition, we'll use the past values of that data to dynamically predict the data's near-future behavior.
```
Comment (austin-schaefer) [inline src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/other-condition-types/create-baseline-alert-conditions.mdx:40]: Double space between `anomaly` and `condition`

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx [hunk 4]
```diff
@@ -277,7 +277,7 @@ Some elements of NRQL used in charts don’t make sense in the streaming context
       </td>
 
       <td>
-        The `COMPARE WITH` clause is used to compare the values for two different time ranges. This type of query is incompatible with NRQL alerting. We recommend using a [Baseline Alert Condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-baseline-alert-conditions) to dynamically detect deviations for a particular signal.
+        The `COMPARE WITH` clause is used to compare the values for two different time ranges. This type of query is incompatible with NRQL alerting. We recommend using a [Anomaly Alert Condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-anomaly-alert-conditions) to dynamically detect deviations for a particular signal.
```
Comment (austin-schaefer) [inline src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx:280]: Did this URL actually change as a result of this sweep? If only the doc title (and not the filename) changed, I don't think this link would've changed, right?

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/introduction-alerts.mdx [hunk 5]
```diff
@@ -125,7 +125,7 @@ Besides the standard controls you'd expect from a complete alerting solution, we
       </td>
 
       <td>
-        [Baseline alert conditions](/docs/alerts/new-relic-alerts/defining-conditions/create-baseline-alert-conditions) allow you to create intelligent, self-adjusting conditions.
+        [Anomaly alert conditions](/docs/alerts/new-relic-alerts/defining-conditions/create-anomaly-alert-conditions) allow you to create intelligent, self-adjusting conditions.
```
Comment (austin-schaefer) [inline src/content/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/introduction-alerts.mdx:128]: See previous comment on URL

File: src/content/docs/apis/nerdgraph/examples/nerdgraph-api-nrql-condition-alerts.mdx [hunk 6]
```diff
@@ -83,16 +83,16 @@ mutation {
...
+## NRQL anomaly condition [#anomaly-condition]
 
-Here's an example of creating a baseline condition:
+Here's an example of creating a anomaly condition:
 
 ```
 mutation {
-  alertsNrqlConditionBaselineCreate(accountId: <var>YOUR_ACCOUNT_ID</var>, policyId: <var>YOUR_POLICY_ID</var>, condition: {
-    name: "Baseline Condition"
+  alertsNrqlConditionAnomalyCreate(accountId: <var>YOUR_ACCOUNT_ID</var>, policyId: <var>YOUR_POLICY_ID</var>, condition: {
```
Comment (austin-schaefer) [inline src/content/docs/apis/nerdgraph/examples/nerdgraph-api-nrql-condition-alerts.mdx:92]: Just wanted to double-check that the condition name has actually been changed in the underlying code. Sometimes the product team cahnges the public-facing language without changing the underlying attribute/event names.

File: src/content/docs/logs/ui-data/find-unusual-logs-log-patterns.mdx [hunk 7]
```diff
@@ -216,4 +216,4 @@ Patterns are a value that is enriched onto the existing log message as a new att
 
 * Build your own [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards) with patterns, to monitor a specific pattern or group of patterns you care about.
 * Create alerts for patterns by adding [NRQL alerts](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions).
-* Use [baseline alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-baseline-alert-conditions) to detect anomalies in known log patterns.
+* Use [anomaly alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-anomaly-alert-conditions) to detect anomalies in known log patterns.
```
Comment (austin-schaefer) [inline src/content/docs/logs/ui-data/find-unusual-logs-log-patterns.mdx:219]: See previous comment re:URL

File: src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/apm-best-practices-guide.mdx [hunk 8]
```diff
@@ -90,7 +90,7 @@ You can manage alerts by creating specific user groups and by leveraging New Rel
 
 See the detailed documentation:
 
-* To set up dynamic baseline alerts and choose an application, [follow standard procedures](/docs/alerts/new-relic-alerts/defining-conditions/create-baseline-alert-conditions). You will see a preview of the metric with the predicted baseline You can select a metric for that application and see the corresponding baseline. Then, using the threshold sliders, you can set how closely you want your threshold to follow the baseline prediction.
+* To set up dynamic anomaly alerts and choose an application, [follow standard procedures](/docs/alerts/new-relic-alerts/defining-conditions/create-anomaly-alert-conditions). You will see a preview of the metric with the predicted anomaly You can select a metric for that application and see the corresponding anomaly. Then, using the threshold sliders, you can set how closely you want your threshold to follow the anomalyprediction.
```
Comment (austin-schaefer) [inline src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/apm-best-practices-guide.mdx:93]: See previous comment re:URL.

Also looks like `anomalyprediction` got run together here

File: src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/browser-monitoring-best-practices-guide.mdx [hunk 9]
```diff
@@ -225,23 +225,23 @@ New Relic provides [unified alerting](/docs/alerts/new-relic-alerts/getting-star
...
+While some metrics can be easily tracked against specific thresholds, other types of data can be more cyclical or have variable ranges for what's considered healthy. Traffic throughput is a good example of this; it can have significantly variable cycles, but large traffic drops or spikes may be important indicators of a breakage preventing user traffic or a DDoS attack that's spiking traffic. [Anomaly alerts](/docs/alerts/new-relic-alerts/defining-conditions/create-anomaly-alert-conditions) can be helpful for creating an expected "band" of normal activity to create more signal to noise in your alerting.
```
Comment (austin-schaefer) [inline src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/browser-monitoring-best-practices-guide.mdx:232]: See previous comment re:URL

File: src/content/docs/new-relic-solutions/new-relic-solutions/optimize-cloud-architecture-spend-continuously-improve-your-modern-cloud-environment.mdx [hunk 10]
```diff
@@ -147,7 +147,7 @@ In the sample dashboard above, the CPU-usage widget on the left reveals that thi
 
 This is just one example of how to identify cloud-based resources that could be candidates for optimization.
 
-Now that you've identified architecture for optimization, go ahead and do so. Whether you right-size your instances, fine tune your databases, modify your storage usage, better configure your load balancers, or even re-architect your applications, in the end your goal is to be able to compare your new, optimized architecture against the baselines you captured in Step 2. For more about baselines please review the [Establish Objectives & Baselines tutorial](/docs/1-adopt-easily-establish-objectives-baselines-define-team-slos).
+Now that you've identified architecture for optimization, go ahead and do so. Whether you right-size your instances, fine tune your databases, modify your storage usage, better configure your load balancers, or even re-architect your applications, in the end your goal is to be able to compare your new, optimized architecture against the anomaly you captured in Step 2. For more about anomaly please review the [Establish Objectives & Anomaly tutorial](/docs/1-adopt-easily-establish-objectives-anomalys-define-team-slos).
```
Comment (austin-schaefer) [inline src/content/docs/new-relic-solutions/new-relic-solutions/optimize-cloud-architecture-spend-continuously-improve-your-modern-cloud-environment.mdx:150]: Should this be `For more about anomalies`?

Also, did this URL change?

File: src/content/docs/new-relic-solutions/observability-maturity/customer-experience/customer-experience-quality-foundation-guide.mdx [hunk 11]
```diff
@@ -503,7 +503,7 @@ Find the most common failures:
 5. Remove third party errors that do not add value.
 
 You may be using a third party JavaScript that is noisy but works as expected. You can take a couple of approaches:
-* Remove the domain name from the JavaScript error/Pageview ratio widget and add it as its own widget so you can see unexpected changes. You can alert on this using [Baseline NRQL](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-baseline-alert-conditions/) alerts.
+* Remove the domain name from the JavaScript error/Pageview ratio widget and add it as its own widget so you can see unexpected changes. You can alert on this using [Anomaly NRQL](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-anomaly-alert-conditions/) alerts.
```
Comment (austin-schaefer) [inline src/content/docs/new-relic-solutions/observability-maturity/customer-experience/customer-experience-quality-foundation-guide.mdx:506]: Did this URL change?

File: src/content/docs/new-relic-solutions/overview.mdx [hunk 12]
```diff
@@ -21,5 +21,5 @@ Browse at your pace and learn with these resources:
 
 * **Observability maturity**: [Our observability maturity practice](/docs/new-relic-solutions/observability-maturity/introduction) identifies common needs and teaches you how to meet them. We have implementation guides, reference guides, self paced training, and hands-on labs.
 * **Best practices guides**: [Learn best practices for some key New Relic features](/docs/new-relic-solutions/best-practices-guides).
-* **Cloud migration solutions**: Moving to the cloud? Learn how to [create application baselines](/docs/new-relic-solutions/new-relic-solutions/create-application-baselines) and [optimize your architecture and spend](/docs/new-relic-solutions/new-relic-solutions/optimize-cloud-architecture-spend-continuously-improve-your-modern-cloud-environment).
+* **Cloud migration solutions**: Moving to the cloud? Learn how to [create application anomalies](/docs/new-relic-solutions/new-relic-solutions/create-application-anomalies) and [optimize your architecture and spend](/docs/new-relic-solutions/new-relic-solutions/optimize-cloud-architecture-spend-continuously-improve-your-modern-cloud-environment).
```
Comment (austin-schaefer) [inline src/content/docs/new-relic-solutions/overview.mdx:24]: Did this URL change?

File: src/nav/alerts-and-applied-intelligence.yml [hunk 13]
```diff
@@ -29,8 +29,8 @@ pages:
             path: /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
       - title: Detect anomalies
         pages:
-          - title: Proactive detection
-            path: /docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence
+          - title: Anomaly detection
```
Comment (austin-schaefer) [inline src/nav/alerts-and-applied-intelligence.yml:32]: Okay now i get to the URL. :D I think the file itself will have to be renamed for this path to change (and in that case you'd also want to add a redirect)

**Review by austin-schaefer — APPROVED** [review]
Approved, with some copy edits and possible URL redirect issues

File: src/content/docs/apis/nerdgraph/examples/nerdgraph-api-nrql-condition-alerts.mdx (hunk 6 above)
Comment (homelessbirds) [reply src/content/docs/apis/nerdgraph/examples/nerdgraph-api-nrql-condition-alerts.mdx:92]: @homelessbirds underlying code was not changed

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx [hunk 14]
```diff
@@ -12,7 +12,7 @@ redirects:
   - /docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence
 ---
 
-With applied intelligence's proactive detection, anomalies from your APM-monitored applications are automatically surfaced in our activity streams and anomalies feed. You can click each anomaly to bring up an automatic analysis.
+With applied intelligence's anomlay detection, anomalies from your APM-monitored applications are automatically surfaced in our activity streams and anomalies feed. You can click each anomaly to bring up an automatic analysis.
```
Comment (rr0214) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx:15]: Do Not change "proactive detection" to "anomaly detection". We'll do a separate effort to update the documentation on proactive detection.  Please leave all proactive detection files as is. This is a separate product.

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/other-condition-types/create-anomaly-alert-conditions.mdx [hunk 15]
```diff
@@ -26,92 +27,92 @@ import baselineOneFacet from 'images/baseline-one-facet.png'
...
+* Use the slider bar to adjust the <img title="Alerts v3: Critical alert icon" alt="icon-alert-critical.png" src={iconAlertCritical}/> **Critical** threshold sensitivity, represented in the preview chart by the light gray area around the anomaly. The tighter the band around the anomaly, the more sensitive it is and the more violations it will generate.
+* Optional: You can create a <img title="Alerts v3: Warning alert icon" alt="icon-alert-warning.png" src={iconAlertWarning}/> [**Warning** threshold](/docs/alerts/new-relic-alerts/defining-conditions/set-thresholds-alert-condition#threshold-levels) (the darker gray area around the anomaly).
 * For NRQL alerts, see the [allowed types of NRQL queries](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries#threshold-types).
 * If the alert condition applies to multiple apps, you can select a choice from the dropdown above the chart to use different metrics. (Not applicable for NRQL alert conditions.)
 
-## Faceted baseline conditions [#baseline-facets]
+## Faceted anomaly conditions [#anomaly-facets]
```
Comment (bcomptonNR) [inline src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/other-condition-types/create-anomaly-alert-conditions.mdx:80]: Double check that all these links are addressed with this change - https://github.com/newrelic/docs-website/search?q=%23baseline-facets

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx]: fixed this!

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/set-thresholds-alert-condition.mdx (hunk 2 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/set-thresholds-alert-condition.mdx]: fixed

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx (hunk 14 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx]: fixed!

File: src/content/docs/new-relic-solutions/new-relic-solutions/optimize-cloud-architecture-spend-continuously-improve-your-modern-cloud-environment.mdx (hunk 10 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-solutions/optimize-cloud-architecture-spend-continuously-improve-your-modern-cloud-environment.mdx]: fixed!

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/other-condition-types/create-anomaly-alert-conditions.mdx (hunk 15 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/other-condition-types/create-anomaly-alert-conditions.mdx:80]: fixed!

**Pull request merged**